app/templates/events/trackVolunteers.html

```diff
@@ -98,7 +98,7 @@ <h5 class="modal-title" id="exampleModalLabel">Add a volunteer</h5>
         <td>{{participant.user.email}}</td>
         <td>{{participant.user.phoneNumber}}</td>
 
-        <td>{% if participant.rsvp == 1 %} Yes {% else %} No {% endif %}</td>
+        <td>{% if participant.user in eventRsvpUsers  %} Yes {% else %} No {% endif %}</td>
```

Comment:
Doesn't this whole participant bit need to be changed? There won't be any participants here until the event actually happens, which makes it useless for adding volunteers to an event beforehand. It's fine if we're going to make a new ticket to do that work, but we should note it somewhere.

Reply:
Note: A possibility of adding the EventRsvp people at the Track Volunteer table.